paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/SavedPaymentMethodTab.kt

```diff
@@ -280,32 +276,6 @@ private fun ModifyBadge(
     )
 }
 
-@Composable
```

Comment:
moved this to its own file -- I didn't want vertical mode code to depend directly on tabs